backend/routers/projects.router.js

```diff
@@ -1,81 +1,38 @@
 const express = require("express");
 const router = express.Router();
 
-const Project = require("../models/project.model");
+const { ProjectController } = require('../controllers');
+const { verifyUser } = require('../middleware');
 
-// GET /api/projects/
```

Comment:
Please add one top comment for all routers where you deleted comments like this `// GET /api/projects/`
For example for this router, it can be `// The root is /api/projects`.
Otherwise, it will be a little bit confusing for all newcomers or inexperienced developers.